onnxruntime/core/providers/cpu/tensor/split.cc

```diff
@@ -97,6 +98,8 @@ Status Split::Compute(OpKernelContext* context) const {
     status = ComputeImpl<int32_t>(*context, input);
   else if (input.IsDataType<int64_t>())
     status = ComputeImpl<int64_t>(*context, input);
+  else if (input.IsDataType<uint8_t>())
```

Comment:
Need to add a test for uint8_t to split_op_test.cc?

Reply:
Will add.